src/aa_remove_data/pb_utils.py

```diff
@@ -0,0 +1,164 @@
+from datetime import datetime, timedelta
+from os import PathLike
+
+from aa_remove_data.generated import EPICSEvent_pb2
+
+
+class PBUtils:
+    def __init__(self, filepath: PathLike | None = None):
```

Comment:
filepath here is optional, but I don't know how useful this class is without it. I would suggest making this a required input and then assigning it to a member variable (self.filepath). That way in the subsequent member functions you can call `self.filepath` rather than taking the filepath in as an input.

An additional function called 'update_filepath' or similar can also be added to change the target PB file on the fly. This way we are only using the API provided by PBUtils.

Reply:
There's a few different filepaths, i.e one to read from, one to write to .txt, one to write to .pb. Should all of these be a member variable?